app/routes.py

```diff
@@ -1,2 +1,256 @@
+
+@videos_bp.route("", methods=["GET"])
+def videos_index():
+    videos = Video.query.all()
+    if videos == None:
+        return [], 200
+    else:
+        videos_response = []
+        for video in videos:
+            videos_response.append(video.to_json_video())
+        return jsonify(videos_response), 200
+
+
+@videos_bp.route("", methods = ["POST"])
+def videos():
+    try:
+        request_body = request.get_json()
+        new_video = Video(title =request_body["title"],
+                        release_date =request_body["release_date"],
+                        total_inventory = request_body["total_inventory"])
+
+        db.session.add(new_video)
+        db.session.commit()
+
+        return jsonify({"id": new_video.video_id}), 201
+    except KeyError:
+        return jsonify({
+            "details": "Invalid data"}), 400
+
+# =========================== Rental Routes ========================
+
+# @rentals_bp.route("/<video_id>", methods = ["GET", "PUT", "DELETE"])
+# def get_single_video(video_id):
+#     video = Video.query.get(video_id)
+#     request_body = request.get_json()
+#     if video is None:
+#         return jsonify({"details": "Video does not exist"}), 404
+#     elif request.method == "GET":
+#         return jsonify(video.to_json_video()), 200
+#     elif video.title is None or video.title is not str(video.title):
+#         return jsonify({"details": "Title data is missing"}), 400
+#     elif video.release_date is None:
+#         return jsonify({"details": "Release_date data is missing"}), 400
+#     elif video.total_inventory is None:
+#         return jsonify({"details": "Total_inventory data is missing"}), 400
+#     elif request.method == "PUT":
+#         if all(keys in request_body for keys in ("title","release_date","total_inventory")) == True:
+#             video.title = request_body["title"]
+#             video.release_date = request_body["release_date"]
+#             video.total_inventory = request_body["total_inventory"]
+
+#             db.session.add(video)
+#             db.session.commit()
+#             return jsonify(video.to_json_video()), 200
+#         else:
+#             return jsonify({"error": "Bad Request"}), 400
+#     elif request.method == "DELETE":
+#         db.session.delete(video)
+#         db.session.commit()
+#         return { "id" : video.video_id }, 200
+
+# @videos_bp.route("", methods=["GET"])
+# def videos_index():
+#     videos = Video.query.all()
+#     if videos == None:
+#         return [], 200
+#     else:
+#         videos_response = []
+#         for video in videos:
+#             videos_response.append(video.to_json_video())
+#         return jsonify(videos_response), 200
+
+
+@rentals_bp.route("/check-out", methods = ["POST"])
+def rentals_check_out():
+
+    if request.method == "POST":
+        request_body = request.get_json()
+        request_body_customer = request_body.get('customer_id')
+        request_body_video = request_body.get('video_id')
```

Comment:
right here we want to query the specific customer and video. This will allow line 187 to work. Right now, we are grabbing the value of the key "customer_id", which is a number, not the entire customer.

```suggestion
        request_body_customer = Customer.query.get(request_body['customer_id'])
        request_body_video = Video.query.get(request_body['video_id'])
```